lib/ui/homePage(temporary).dart

```diff
@@ -27,7 +27,7 @@ class _MyHomePageState extends State<MyHomePage> {
   void showSnackBar() {
     _scaffoldKey.currentState.showSnackBar(SnackBar(
       content: Text(
-        "Welcome, ${widget.name.substring(9)}",
+       (widget.name!=null)? "Welcome, ${widget.name.substring(9)}":"Welcome Back!",
```

Comment:
🤔🤔

Reply:
This is for snackbar and it will be removed in new design(I think).